src/CSVReader.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Created by ken-linux2 on 14/07/16.
+ */
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+
+public class CSVReader {
+    public static void main(String[] args) {
+        CSVReader RCF = new CSVReader();
+        RCF.run();
+    }
+
+    private void run() {
+        final String csvFile = "/home/ken-linux2/Downloads/macycExampleFile.csv - macycExampleFile.csv.csv";
+        BufferedReader br = null;
+        String line;
+        String str;
+        String cvsSplitBy = ",";
+
+        try {
+            br = new BufferedReader(new FileReader(csvFile));
+            while ((line = br.readLine()) != null) {
+                final String[] myFile = line.split(cvsSplitBy);
+                for (String item:myFile) {
+                    str= item+("\t");
+                    System.out.print(str);
+                }
+               System.out.println();
+        }
+
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        } catch (IOException e) {
+            e.printStackTrace();
+        } finally {
```

Comment:
You want me to implement the closable Interface?


On Tue, Jul 19, 2016 at 2:13 PM, Meytal Traub <notifications@github.com>
wrote:

> In src/CSVReader.java
> <https://github.com/NirHershkovitz/corejavaexercise/pull/3#discussion_r71318762>
> :
>
> > +        try {
> > +            br = new BufferedReader(new FileReader(csvFile));
> > +            while ((line = br.readLine()) != null) {
> > +                final String[] myFile = line.split(cvsSplitBy);
> > +                for (String item:myFile) {
> > +                    str= item+("\t");
> > +                    System.out.print(str);
> > +                }
> > +               System.out.println();
> > +        }
> > +
> > +        } catch (FileNotFoundException e) {
> > +            e.printStackTrace();
> > +        } catch (IOException e) {
> > +            e.printStackTrace();
> > +        } finally {
>
> don't forget to check about closable in java 7
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/NirHershkovitz/corejavaexercise/pull/3/files/3a201adb7ac8b1a5de01036b9917e0b5835b0089#r71318762>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ATczEkwoxHdfvUjQgv_LE8zCN-zQPUJIks5qXLFOgaJpZM4JOmV5>
> .
>

-- 
This e-mail, as well as any attached document, may contain material which 
is confidential and privileged and may include trademark, copyright and 
other intellectual property rights that are proprietary to Kenshoo Ltd, 
 its subsidiaries or affiliates ("Kenshoo"). This e-mail and its 
attachments may be read, copied and used only by the addressee for the 
purpose(s) for which it was disclosed herein. If you have received it in 
error, please destroy the message and any attachment, and contact us 
immediately. If you are not the intended recipient, be aware that any 
review, reliance, disclosure, copying, distribution or use of the contents 
of this message without Kenshoo's express permission is strictly prohibited.